WordPress/src/test/java/org/wordpress/android/ui/domains/management/details/DomainManagementDetailsWebViewNavigationDelegateTest.kt

```diff
@@ -26,6 +25,8 @@ class DomainManagementDetailsWebViewNavigationDelegateTest : BaseUnitTest() {
     fun `when browsing outside the domains path, then the web view cannot navigate`() {
         Assertions.assertThat(
             buildUrls(
+                "/domains/manage/all/some.domain/dns/some.slug", // standard details page
```

Comment:
Is this the standard details page, or is the code comment possibly a copy / paste error?

Reply:
Good catch Matthew 👍 

> is the code comment possibly a copy / paste error?

Exactly 😓. Fixed with 3b6145b8429927237c6f218ac5aa7e61e9b3baa3